src/app/view/perspectives/search/documents/search-documents.component.scss

```diff
@@ -0,0 +1,18 @@
+@import "../../../../shared/variables";
+
+/deep/.search-documents-attribute {
```

Comment:
I think we should avoid using `deep` selector. Why does it need to be used here?

Reply:
html is pasted in as a string.
Angular ignores the class unless the _ngcontext matches.

Reply:
And why do we put there HTML from string? Cannot it be directly in the template?

Reply:
Probably. (I guess the recursive nature of some values was the reason @kubedo8 made it like that)
It is out of the scope of this PR, so I'll leave at that.

Reply:
@justmesr yes you are right. Document may have nested attributes and values and generating html directly in the template was very slow even for document with few attributes.